### ISSUE TITLE
How to pass reporter param to gradle task

### ISSUE THREAD

**Issue by CoXier** [shyiko/ktlint#216]
As titles says, how to pass reporter param to gradle task  `ktlint`?

>  // to generate report in checkstyle format prepend following args:
> //     "--reporter=plain", "--reporter=checkstyle,output=${buildDir}/ktlint.xml"

Your comment in the task `ktlint` is a little difficult for me to understand.

**Comment by shyiko** [comment]
e.g.
```groovy
task ktlint(type: JavaExec, group: "verification") {
    description = "Check Kotlin code style."
    classpath = configurations.ktlint
    main = "com.github.shyiko.ktlint.Main"
    args "--reporter=plain?group_by_file", "src/**/*.kt"
}
```
